pages/slice-simulator.jsx

```diff
@@ -1,7 +1,7 @@
 import { SliceSimulator } from '@prismicio/slice-simulator-react'
 import { SliceZone } from '@prismicio/react'
 
-import { components } from '../slices'
+// import { components } from '../slices.'
```

Comment:
Please delete it if it is unnecessary, or add further explanation why you've commented this out.